cmd/operator-sdk/olmcatalog/gen-csv.go

```diff
@@ -32,10 +32,13 @@ import (
 )
 
 var (
-	csvVersion    string
-	fromVersion   string
-	csvConfigPath string
-	updateCRDs    bool
+	csvVersion     string
+	fromVersion    string
+	csvConfigPath  string
+	updateCRDs     bool
+	absProjectPath = projutil.MustGetwd()
```

Comment:
Revert these globals back to their original spots.

Reply:
done